### ISSUE TITLE
Rate Limited

### ISSUE THREAD

**Issue by NinjaLukeI** [char/blm-instagram-bot#14]
Not sure if this issue is solely with my implementation but for some reason I am constantly being given the "rate limited" error message, along with an index error from the line of code "post = feed['items'].pop(0).

**Comment by mathematicalmichael** [comment]
yup that's to do with some of the implementation choices with how exceptions are caught
Sounds like you need to create a copy of `.env.example` with the name `.env`.
